_studio/shared/umc/codec/h265_dec/include/umc_h265_bitstream_headers.h

```diff
@@ -549,7 +549,12 @@ inline size_t H265BaseBitstream::BytesLeft() const
 // Throw exception if left bits not enough to decode
 inline void H265BaseBitstream::CheckBitsLeft(const uint32_t nbits)
 {
-    if ((int32_t)m_maxBsSize * 8 - BitsDecoded() < nbits)
+    if((int32_t)m_maxBsSize > 0 && (size_t)m_maxBsSize * 8 > BitsDecoded())
```

Comment:
when will m_maxBsSize be less then 0?

Reply:
m_maxBsSize will be recalculated when AllocFrames, It is uncertain whether the value will be changed to negative due to the user's hacker operation, therefore, protection is carried out in advance.
Just as BitsDecoded () here returns an abnormal value, when the user decodes with a manually customized stream, it may result in an abnormal assignment, resulting in an out-of-bounds read operation(which we want to avoid).

Reply:
I still can't get it. 
Anyway, approved.